### ISSUE TITLE
status badges/icons in dolphin integration

### ISSUE THREAD

**Issue by classabbyamp** [Martchus/syncthingtray#151]
**Relevant components**
* [ ] Standalone tray application (based on Qt Widgets)
* [ ] Plasmoid/applet for Plasma desktop
* [x] Dolphin integration
* [ ] Command line tool (`syncthingctl`)
* [ ] Integrated Syncthing instance (`libsyncthing`)
* [ ] Backend libraries

**Describe the solution you'd like**

Something like this (the status icons/badges provided by [dolphin plugins](https://apps.kde.org/dolphin_plugins/) for syncthingtray's dolphin plugin

![image](https://user-images.githubusercontent.com/5366828/184051418-57ff55f0-c82b-44d5-a17b-1a0ae3d3aef7.png)

**Comment by Martchus** [comment]
That would be very nice, although last time I've looked into it I wasn't sure what Syncthing API to use.

**Comment by Martchus** [comment]
Yes. There are so many different things to improve right now so likely I won't have time to look into anytime soon.

**Comment by classabbyamp** [comment]
right, I'm just dismissing the stale bot

**Comment by chainria** [comment]
So, is there any possibility to get this added? Is there still trouble with finding out the right API calls?
If so, I can try looking into them.

**Comment by Martchus** [comment]
I haven't had the time to look into it. If this is still wanted it would make most sense if you looked into it yourself.

**Comment by chainria** [comment]
Thank you for the quick response!

Well, here's the thing. I came from Nextcloud files to Syncthing, because it's just more suited to my usecase.
I am used to the badges and was shocked to not finding them anywhere. I just assumed they were there.
I am pretty sure I'm not the only one thinking like this. So yeah, I will look into it and report my findings.
At least I hope there even *is* an API call to find the state of a file...
